lms/djangoapps/grades/management/commands/tests/test_compute_grades.py

```diff
@@ -59,6 +60,18 @@ def test_selecting_invalid_course(self):
         with self.assertRaises(CommandError):
             self.command._get_course_keys({'courses': [self.course_keys[0], self.course_keys[1], 'badcoursekey']})
 
+    def test_from_settings(self):
+        ComputeGradesSetting.objects.create(course_ids=self.course_keys[0] + " " + self.course_keys[1])
```

Comment:
In python, connecting strings with + is a bad idea, because the python interpreter allocates memory for a new string for every plus.  It ends up creating a number of intermediate strings that waste time and memory.  Instead, do `" ".join([self.course_keys[0], self.course_keys[1]])` or the equivalent `" ".join(self.course_keys[:2])` when joining a list together, or use string formatting to create more structured strings.